tests/driver_ds3231/main.c

```diff
@@ -28,15 +28,13 @@
 #include "ds3231.h"
 #include "ds3231_params.h"
 
+#define DS3231_FLAG_ALARM_1 (0x01)
```

Comment:
That should be in the driver header, along with one for `DS3231_FLAG_ALARM_2` and the information that it's returned by `ds3231_await_alarm`

Reply:
Yes that makes a lot more sense.